### ISSUE TITLE
build instructions?

### ISSUE THREAD

**Issue by dozens** [vinnymac/PokeNurse#111]
Is there any documentation on how to build? I love the tool and getting impatient waiting for the next release :) so I would like to build myself (for mac osx if it matters).

**Comment by mackhankins** [comment]
checkout the develop branch.  I think we're waiting on the resolutions of #102 for 1.2

Edit: https://github.com/vinnymac/PokeNurse/tree/develop

**Comment by dozens** [comment]
Are there individual targets for each OS, building eveything is taking ALOT of disk space.

**Comment by mackhankins** [comment]
`npm run package` just builds your OS

**Comment by dozens** [comment]
My dev environment is linux and I want to build osx.

**Comment by vinnymac** [comment]
Code on develop is buggy and potentially dangerous. I cannot recommend ever building on that branch. We may make it easier for people to make builds for any OS again, but it is not a priority. If you must do this you can edit the package.js to only build specific os'.
